src/components/users/users.ts

```diff
@@ -143,38 +140,38 @@ async function setAllUserRolesForOrg(
 
   await Promise.all(
     spaces.map((space: ISpace) => spaceRoleEndpoints.map(async (role: string) => {
-      /* istanbul ignore next */
-      if (!roles.space[space.metadata.guid]) {
-        return Promise.resolve(undefined);
-      }
-
-      const oldPermission = roles.space[space.metadata.guid][role].current;
-      const newPermission = roles.space[space.metadata.guid][role].desired;
-
-      if (newPermission && newPermission === oldPermission) {
-        return Promise.resolve(undefined);
-      }
-
-      if (isUndefined(newPermission) && oldPermission === '0') {
-        return Promise.resolve(undefined);
-      }
-
-      return cf.setSpaceRole(
-        space.metadata.guid,
-        params.userGUID,
-        role,
-        newPermission === '1',
-      );
-    }),
-  ));
+        /* istanbul ignore next */
+        if (!roles.space[space.metadata.guid]) {
+          return Promise.resolve(undefined);
+        }
+
+        const oldPermission = roles.space[space.metadata.guid][role].current;
+        const newPermission = roles.space[space.metadata.guid][role].desired;
+
+        if (newPermission && newPermission === oldPermission) {
+          return Promise.resolve(undefined);
+        }
+
+        if (isUndefined(newPermission) && oldPermission === '0') {
+          return Promise.resolve(undefined);
+        }
+
+        return cf.setSpaceRole(
+          space.metadata.guid,
+          params.userGUID,
+          role,
+          newPermission === '1',
+        );
+      }),
+    ));
```

Comment:
A lot of the formatting changes in this PR make sense, but here you're changing something from 2-space indentation to 4-space indentation. Is that intentional?

Reply:
@AP-Hunt tells me this was autoformatted to comply with `tslint`. Resolving…